src/java/voldemort/client/protocol/admin/AdminClient.java

```diff
@@ -1156,20 +1210,12 @@ private void internalUpdateMetadataversion(Collection<String> versionKeys) {
          * @param newProperties The new metadata versions to be set across all
          *        the nodes in the cluster
          */
-        public void setMetadataversion(Properties newProperties) {
-            MetadataVersionStoreUtils.setProperties(AdminClient.this.metadataVersionSysStoreClient,
-                                                    newProperties);
-        }
-
-        /**
-         * Set the metadata versions to the given set
-         * 
-         * @param newProperties The new metadata versions to be set across all
-         *        the nodes in the cluster
-         */
         public void setMetadataversion(Versioned<Properties> newProperties) {
```

Comment:
Should we refactor this so that the camel casing is consistent across all overloads of this function? (i.e.: ```setMetadataVersion```?)

I think it would also be helpful to move this function next to its other overloads. It's easier to reason about an API when all of the overloads are grouped together...